File: Week6/sentimental-cash/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height = 0;

    do
    {
        height = get_int("Height: ");

    }

    while (height < 1 || height > 8);
    if(height < 1 & height > 8){
        return 0;
    }
    for (int i = 0; i < height; i++)
    {
        for (int j = 1; j < height - i; j++)
        {
            printf(" ");
        }
        for (int x = -1; x < i; x++)
        {
            printf("#");
        }
        // printf("  ");

        printf("\n");
    }
}

File: Week6/sentimental-mario-less/mario.py
from cs50 import get_int

s = 0
while (s < 1 or s > 8):
    s = get_int("Height: ")

blankspaces = s
hashes = 0
for i in range(s):
    blankspaces -= 1
    hashes += 1
    print(" " * blankspaces, end="")
    print("#" * hashes, end="")
    print("")
